### PR TITLE
Update bc-government-organizations-in-github.md

### PR DESCRIPTION
**Pull request by Shivanganii**
For post-cut SSO deadline cut-off

### CODE REVIEW

File: docs/use-github-in-bcgov/bc-government-organizations-in-github.md [hunk 1]
```diff
@@ -31,7 +31,6 @@ The `bcgov` organization contains public repositories that hold open-source code
 
 * Product teams working in the B.C. Gov Private Cloud PaaS OpenShift platform should use this organization unless they are given permission to work in [`bcgov-c`](#bcgov-c).
 * Any member of the `bcgov` organization can create repositories.
-* Existing members of the `bcgov` organization can invite new users to join the organization using the [Just Ask! tool](https://just-ask.developer.gov.bc.ca/). (***Users requesting membership to `bcgov` for themselves will be reviewed and may be declined***)
```
Comment (MonicaG) [inline docs/use-github-in-bcgov/bc-government-organizations-in-github.md:34]: Should there be instructions here on how to join the bcgov org? I think we would want similar instructions as what was put into [Update README.md](https://github.com/bcgov/.github/pull/5). Maybe we want the instructions in that README to point to the instructions here instead?

File: docs/use-github-in-bcgov/bc-government-organizations-in-github.md [hunk 2]
```diff
@@ -43,66 +42,16 @@ The `bcgov-c` organization stores temporary (up to 12 months), private repositor
...
- 
-That’s it, your accounts are now linked.  
  
-### Directions to link your account after July 30 
+### Directions to link your account
 
-Employees and contractors locked out after **July 30th, 2024** can follow the steps below to regain access. If you are a contractor without an IDIR, follow the steps in the [IDIR transition guide](github-transition-guide.md#idirs) before proceeding:
+If you are a contractor without an IDIR, follow the steps in the [IDIR transition guide](github-transition-guide.md#idirs) before proceeding:
  
 1. If you have a valid IDIR, ask anyone with access to the organization to re-invite you.
```
Comment (MonicaG) [inline docs/use-github-in-bcgov/bc-government-organizations-in-github.md:56]: This should be self serve now and shouldn't require someone else inviting them in.  Maybe something like: 
```suggestion
1. If you have a valid IDIR, follow the instructions at (https://developer.gov.bc.ca/docs/default/component/bc-developer-guide/use-github-in-bcgov/bc-government-organizations-in-github/#bcgov) to join the bcgov organization.
```

File: docs/use-github-in-bcgov/bc-government-organizations-in-github.md [hunk 3]
```diff
@@ -43,66 +42,16 @@ The `bcgov-c` organization stores temporary (up to 12 months), private repositor
...
     - [Learn how to request access to the bcgov GitHub organization](https://developer.gov.bc.ca/docs/default/component/bc-developer-guide/use-github-in-bcgov/bc-government-organizations-in-github/#bcgov).
```
Comment (MonicaG) [inline docs/use-github-in-bcgov/bc-government-organizations-in-github.md:57]: This probably isn't needed given the above suggestion

**Review by MonicaG — CHANGES_REQUESTED** [review]
Hi @Shivanganii  - I left some comments. Please let me know if you would like to discuss them. 😃

File: docs/use-github-in-bcgov/bc-government-organizations-in-github.md [hunk 4]
```diff
@@ -43,66 +42,16 @@ The `bcgov-c` organization stores temporary (up to 12 months), private repositor
...
-We enabled single sign-on (SSO) for GitHub. We’re starting with the [bcgov GitHub organization](https://github.com/bcgov). After we finish transitioning [bcgov](#bcgov), we’ll enable other OCIO-managed GitHub organizations. 
-  
-To keep access to the bcgov GitHub organization, all users must link their IDIRs to their GitHub IDs by **July 30, 2024**. If users don’t complete this by the deadline, they’ll be automatically removed from the bcgov GitHub organization including all the code repositories connected to [bcgov](#bcgov).   
-
-Contractors without IDIRs must order one before SSO can be enabled. 
+To keep access to the bcgov GitHub organization, all users must be authenticated. Contractors without IDIRs must order one before SSO can be enabled. 
 
 [Learn more about IDIRs](https://dev.developer.gov.bc.ca/docs/default/component/bc-developer-guide/use-github-in-bcgov/github-transition-guide/). 
- 
-[Learn more about the timelines and instructions](bc-government-organizations-in-github.md#the-changes).
```
Comment (jenreiher-bcgov) [inline docs/use-github-in-bcgov/bc-government-organizations-in-github.md:56]: we'll need some version of this, but lighter, for the migration to the bcgov-c GitHub organization. we should have dates for this by next week.

Comment (Shivanganii) [reply docs/use-github-in-bcgov/bc-government-organizations-in-github.md:56]: I re-purposed the unrefined instructions and changes text the bcgov-c part for now, and once dates, etc. is finalized, I'll re-work the text.

Comment (Shivanganii) [reply docs/use-github-in-bcgov/bc-government-organizations-in-github.md:56]: @jenreiher-bcgov  I re-purposed the unrefined instructions and changes text the bcgov-c part for now, and once dates, etc. is finalized, I'll re-work the text.

File: docs/use-github-in-bcgov/bc-government-organizations-in-github.md [hunk 5]
```diff
@@ -43,68 +42,18 @@ The `bcgov-c` organization stores temporary (up to 12 months), private repositor
...
+## Single sign-on 
 
-We enabled single sign-on (SSO) for GitHub. We’re starting with the [bcgov GitHub organization](https://github.com/bcgov). After we finish transitioning [bcgov](#bcgov), we’ll enable other OCIO-managed GitHub organizations. 
-  
-To keep access to the bcgov GitHub organization, all users must link their IDIRs to their GitHub IDs by **July 30, 2024**. If users don’t complete this by the deadline, they’ll be automatically removed from the bcgov GitHub organization including all the code repositories connected to [bcgov](#bcgov).   
-
-Contractors without IDIRs must order one before SSO can be enabled. 
+To keep access to the bcgov GitHub organization, all users must be authenticated. Contractors without IDIRs must order one before SSO can be enabled. 
 
 [Learn more about IDIRs](https://dev.developer.gov.bc.ca/docs/default/component/bc-developer-guide/use-github-in-bcgov/github-transition-guide/).
```
Comment (jenreiher-bcgov) [inline docs/use-github-in-bcgov/bc-government-organizations-in-github.md:49]: just noticed this is linking to the wrong thing! it's linking to the draft PR site. And, please use a tracking link.

File: docs/use-github-in-bcgov/bc-government-organizations-in-github.md (hunk 1 above)
Comment (Shivanganii) [reply docs/use-github-in-bcgov/bc-government-organizations-in-github.md:34]: > Should there be instructions here on how to join the bcgov org? I think we would want similar instructions as what was put into [Update README.md](https://github.com/bcgov/.github/pull/5). Maybe we want the instructions in that README to point to the instructions here instead?

Hm, I like that idea. That way it's in 1 place rather than another. @jenreiher-bcgov are you alight with that?

Comment (jenreiher-bcgov) [reply docs/use-github-in-bcgov/bc-government-organizations-in-github.md:34]: yes! one place is always better :)

Comment (jenreiher-bcgov) [reply docs/use-github-in-bcgov/bc-government-organizations-in-github.md:34]: althhhouuughhh... do we want the one place to be here, or in the readme? 
let's be thoughtful about this, because having the instructions be in DevHub may help us educate folks on other things they may or may not know about GitHub, vs if they never click through it kind of leaves them learning about it in isolation of GitHub.

File: docs/use-github-in-bcgov/bc-government-organizations-in-github.md (hunk 5 above)
Comment (MonicaG) [reply docs/use-github-in-bcgov/bc-government-organizations-in-github.md:49]: I fixed that in #45. The merge of this PR into main should keep the change in #45

File: docs/use-github-in-bcgov/bc-government-organizations-in-github.md (hunk 1 above)
Comment (MonicaG) [reply docs/use-github-in-bcgov/bc-government-organizations-in-github.md:34]: I vote for the one place to be in the bcdg.

**Comment by Shivanganii** [comment]
@jenreiher-bcgov I re-design the page, could you review for accuracy?

File: docs/use-github-in-bcgov/bc-government-organizations-in-github.md [hunk 6]
```diff
@@ -1,120 +1,96 @@
...
-* Collaborate with the open-source community
 * Integrate automation tools
+* Track issues
 
-The main organization the B.C. government owns in GitHub is [bcgov](https://github.com/bcgov) where we store all open-source code developed by B.C. government teams. The `bcgov` organization includes over 1000 repositories maintained by the B.C. government developer community.
-
-The Developer Experience Team manages the B.C. government's GitHub environments.
-
-## Working in the open
+B.C. Government's main organization is [bcgov](https://github.com/bcgov), where we store all open-source code developed by government teams. The `bcgov` organization includes over 1000 repositories. We also have the `bcgov-c`organization, where developers can either permanently or temporarily store confidential code.
```
Comment (jenreiher-bcgov) [inline docs/use-github-in-bcgov/bc-government-organizations-in-github.md:11]: slight nit here, but teams get hung up on this lingo often enough that we have to have whole meetings about it...
teccchnnicalllyyyyy putting code in the open is not the same as open-source code. 

so we probably should say "open code developed by government teams" and if we wanted to elaborate to say "including projects that have open-source and inner-source contribution models" so that it's more obvious these are not the same things...

File: docs/use-github-in-bcgov/bc-government-organizations-in-github.md [hunk 7]
```diff
@@ -1,120 +1,96 @@
...
-
-The Developer Experience Team manages the B.C. government's GitHub environments.
-
-## Working in the open
+B.C. Government's main organization is [bcgov](https://github.com/bcgov), where we store all open-source code developed by government teams. The `bcgov` organization includes over 1000 repositories. We also have the `bcgov-c`organization, where developers can either permanently or temporarily store confidential code. 
 
-The [Digital Principles for the Government of B.C.](https://www2.gov.bc.ca/gov/content/governments/policies-for-government/core-policy/policies/im-it-management#12.1.1.5) urge product teams to work in the open as outlined in principle five. GitHub is the leading platform for open-source projects. It allows the Province to collaborate with the open-source community to build software, support innovation and save time and money.
+On behalf of the B.C. Government, the Developer Experience Team manages government GitHub environments.
```
Comment (jenreiher-bcgov) [inline docs/use-github-in-bcgov/bc-government-organizations-in-github.md:13]: we might want to add our contact information for support here, too!

File: docs/use-github-in-bcgov/bc-government-organizations-in-github.md [hunk 8]
```diff
@@ -1,120 +1,96 @@
...
+  graph TD;
+A[GitHub Enterprise Cloud] --> B[bcgov]
+A --> C[bcgov-c]
+```
 
-| Name | Use for... | User Access Requirements | Cost |
-| ---- | ----------- | ----------- | ----- |
-| [bcgov](#bcgov) | Repositories should be hosted in this open repo unless you have strong rationale from your MISO otherwise. | GitHub ID | Free |
-| [bcgov-c](#bcgov-c) | Code you would like to temporarily host in a closed repo, before moving into the bcgov org. | GitHub ID | Covered by OCIO for up to 12 months |
+Both organizations require users to enable [Single Sign-on](## Single sign-on).
```
Comment (jenreiher-bcgov) [inline docs/use-github-in-bcgov/bc-government-organizations-in-github.md:25]: Both organizations require users to enable [Single Sign-on](## Single sign-on) linked to an IDIR account.

File: docs/use-github-in-bcgov/bc-government-organizations-in-github.md [hunk 9]
```diff
@@ -1,120 +1,96 @@
...
 
-The [Digital Principles for the Government of B.C.](https://www2.gov.bc.ca/gov/content/governments/policies-for-government/core-policy/policies/im-it-management#12.1.1.5) urge product teams to work in the open as outlined in principle five. GitHub is the leading platform for open-source projects. It allows the Province to collaborate with the open-source community to build software, support innovation and save time and money.
+On behalf of the B.C. Government, the Developer Experience Team manages government GitHub environments.
 
 ## Organizations in GitHub
 
-The Province owns several GitHub organizations, which are described below.
+We use GitHub Enterprise Cloud, and it hosts the `bcgov` and `bcgov-c` GitHub organizations: 
+
+```mermaid
```
Comment (jenreiher-bcgov) [inline docs/use-github-in-bcgov/bc-government-organizations-in-github.md:19]: nice!! :D 

you still included the png file here - just FYI in case you forgot to delete it!

File: docs/use-github-in-bcgov/bc-government-organizations-in-github.md [hunk 10]
```diff
@@ -1,120 +1,96 @@
...
 ### `bcgov-c`
 
-The `bcgov-c` organization stores temporary (up to 12 months), private repositories with closed-source code and private documents. Closed-source projects must be moved to the `bcgov` organization at the end of the 12 months. This repository is **private**.
-
-* Use this repository if you need a temporary location for code while you collect approvals to make the code public. You must commit to making the code public in the future in order to use this repository.
-* Only the Developer Experience team can create repositories in this organization. You can request a repository by [submitting a request](https://github.com/BCDevOps/devops-requests/issues/new?assignees=MonicaG%2C+oomIRL&labels=github-repo%2C+pending&projects=&template=github_repo_request.md&title=).
+The `bcgov-c` organization stores permanent, private repositories with confidential source code and documents. This repository is **private**.
 
-Your product team can only have a **permanent**, private repository in `bcgov-c` if it is a GitOps repository with ArgoCD manifests. 
+* Use this repository if you need a permanent location for code, or if you're collecting approvals to make the code public.
```
Comment (jenreiher-bcgov) [inline docs/use-github-in-bcgov/bc-government-organizations-in-github.md:38]: collecting approvals could maybe be "working towards"? there's more work involved usually than just approvals unfortunately

File: docs/use-github-in-bcgov/bc-government-organizations-in-github.md [hunk 11]
```diff
@@ -1,120 +1,96 @@
...
 
-## Single sign-on is coming to the B.C. government’s GitHub organizations
-
-We enabled single sign-on (SSO) for GitHub. We’re starting with the [bcgov GitHub organization](https://github.com/bcgov). After we finish transitioning [bcgov](#bcgov), we’ll enable other OCIO-managed GitHub organizations. 
-  
-To keep access to the bcgov GitHub organization, all users must link their IDIRs to their GitHub IDs by **July 30, 2024**. If users don’t complete this by the deadline, they’ll be automatically removed from the bcgov GitHub organization including all the code repositories connected to [bcgov](#bcgov).   
+## Single Sign-On 
 
-Contractors without IDIRs must order one before SSO can be enabled. 
+To keep access to the bcgov GitHub organization, all users must be authenticated. Contractors without IDIRs must order one before SSO can be enabled.
```
Comment (jenreiher-bcgov) [inline docs/use-github-in-bcgov/bc-government-organizations-in-github.md:45]: To keep access to repositories in the bcgov enterprise

File: docs/use-github-in-bcgov/bc-government-organizations-in-github.md [hunk 12]
```diff
@@ -1,120 +1,96 @@
...
+    - [Learn more about how to do this](github-transition-guide.md#resetting-github-keys). 
  
-Ministries won’t be charged for upgraded GitHub licenses.  However, ordering IDIRs for contractors may cost and need expense approvals. We have more information about IDIRS on our [IDIR transition guide](github-transition-guide.md#idirs). 
+### Upgrading bcgov-c 
+
+
+We begin transition of [`bcgov-c`](#bcgov-c) into Enterprise GitHub on **August 7, 2024**. After migration completes, we'll enable SSO for [`bcgov-c`](#bcgov-c) on **August 21, 2024**, using the same process as the SSO enablement for `bcgov`.
   
-### Directions to link your accounts before July 30
+For current B.C. Government private Enterprise organizations, we’ll work with your respective Information Management Branches and Information Systems Branches to identify next steps
```
Comment (jenreiher-bcgov) [inline docs/use-github-in-bcgov/bc-government-organizations-in-github.md:63]: For current B.C. Government private organizations that are part of the legacy `bcgov-ent` Enterprise implementation, we’ll work with your respective Information Management Branches and Information Systems Branches to identify next steps

File: docs/use-github-in-bcgov/bc-government-organizations-in-github.md [hunk 13]
```diff
@@ -1,120 +1,96 @@
...
-## Working in the open
+B.C. Government's main organization is [bcgov](https://github.com/bcgov), where we store all open-source code developed by government teams. The `bcgov` organization includes over 1000 repositories. We also have the `bcgov-c`organization, where developers can either permanently or temporarily store confidential code. 
 
-The [Digital Principles for the Government of B.C.](https://www2.gov.bc.ca/gov/content/governments/policies-for-government/core-policy/policies/im-it-management#12.1.1.5) urge product teams to work in the open as outlined in principle five. GitHub is the leading platform for open-source projects. It allows the Province to collaborate with the open-source community to build software, support innovation and save time and money.
+On behalf of the B.C. Government, the Developer Experience Team manages government GitHub environments.
 
 ## Organizations in GitHub
 
-The Province owns several GitHub organizations, which are described below.
+We use GitHub Enterprise Cloud, and it hosts the `bcgov` and `bcgov-c` GitHub organizations:
```
Comment (jenreiher-bcgov) [inline docs/use-github-in-bcgov/bc-government-organizations-in-github.md:17]: I wonder if it's worth having some change management language here so folks know this is new/different than what they might have known in the past? a little clue not to skim

File: docs/use-github-in-bcgov/bc-government-organizations-in-github.md [hunk 14]
```diff
@@ -1,120 +1,96 @@
...
  
-[Learn more about how linking works](github-transition-guide.md#technical-information).  
-
-### Costs
+1. Once you have access again, your teammates with admin access may need to re-invite you to the repositories you had access to before.  
+2. Reauthorize your personal access tokens and/or SSH keys so they continue to work with SSO enabled.
+    - [Learn more about how to do this](github-transition-guide.md#resetting-github-keys). 
  
-Ministries won’t be charged for upgraded GitHub licenses.  However, ordering IDIRs for contractors may cost and need expense approvals. We have more information about IDIRS on our [IDIR transition guide](github-transition-guide.md#idirs). 
+### Upgrading bcgov-c
```
Comment (jenreiher-bcgov) [inline docs/use-github-in-bcgov/bc-government-organizations-in-github.md:58]: perhaps this should be `SSO timelines` because it includes info about both bcgov-c and others?

File: docs/use-github-in-bcgov/bc-government-organizations-in-github.md (hunk 6 above)
Comment (Shivanganii) [reply docs/use-github-in-bcgov/bc-government-organizations-in-github.md:11]: Added

File: docs/use-github-in-bcgov/bc-government-organizations-in-github.md (hunk 8 above)
Comment (Shivanganii) [reply docs/use-github-in-bcgov/bc-government-organizations-in-github.md:25]: Yeah...this was a rush-to-sign-off moment. it's not working in preview, so I'll work on this a bit more.

File: docs/use-github-in-bcgov/bc-government-organizations-in-github.md (hunk 9 above)
Comment (Shivanganii) [reply docs/use-github-in-bcgov/bc-government-organizations-in-github.md:19]: Interesting, I tried to delete in a commit, but I suppose that approach didn't work.

File: docs/use-github-in-bcgov/bc-government-organizations-in-github.md (hunk 13 above)
Comment (Shivanganii) [reply docs/use-github-in-bcgov/bc-government-organizations-in-github.md:17]: @jenreiher-bcgov if we go with the version or similar in the comment, I'll remove the last sentence: 

Recently, we enabled Single Sign-On (SSO) for `bcgov` and on **August 21, 2024**, we'll enable SSO for `bcgov-c`. The transition allowed us to understand user needs better, and streamline services.  

Now, we use GitHub Enterprise Cloud, and it hosts the `bcgov` and `bcgov-c` GitHub organizations: 

```mermaid
  graph TD;
A[GitHub Enterprise Cloud] --> B[bcgov]
A --> C[bcgov-c]
```

File: docs/use-github-in-bcgov/bc-government-organizations-in-github.md [hunk 15]
```diff
@@ -1,120 +1,102 @@
 # B.C. Government organizations in GitHub
 
-Many teams building software for the B.C. government use [GitHub](https://github.com) to host their code. Using GitHub you can:
+The [Digital Principles for B.C. Government](https://www2.gov.bc.ca/gov/content/governments/policies-for-government/core-policy/policies/im-it-management#12.1.1.5) urge product teams to work in the open. GitHub is the leading platform for open-source projects. It lets our organization to work with the open-source community to build software, support innovation and save time and money.
```
Comment (jenreiher-bcgov) [inline docs/use-github-in-bcgov/bc-government-organizations-in-github.md:3]: the last sentence here has some grammatical curiosities

File: docs/use-github-in-bcgov/bc-government-organizations-in-github.md [hunk 16]
```diff
@@ -1,120 +1,102 @@
...
-The `bcgov` organization contains public repositories that hold open-source code or public documents developed by B.C. government teams. This organization is **public**.
+The `bcgov` organization contains public repositories that hold open-source code or public documents developed by B.C. government teams. This organization is **public**.  To align with our Digital Principles, repositories should be hosted in this open repo. The only exception is strong rationale from your Ministry Security Information Officer.
 
-* Product teams working in the B.C. Gov Private Cloud PaaS OpenShift platform should use this organization unless they are given permission to work in [`bcgov-c`](#bcgov-c).
-* Any member of the `bcgov` organization can create repositories.
-* Existing members of the `bcgov` organization can invite new users to join the organization using the [Just Ask! tool](https://just-ask.developer.gov.bc.ca/). (***Users requesting membership to `bcgov` for themselves will be reviewed and may be declined***) 
+* Product teams working in the B.C. Government Private Cloud PaaS OpenShift platform should use this organization. The exception is permission to work in [`bcgov-c`](#bcgov-c)
```
Comment (jenreiher-bcgov) [inline docs/use-github-in-bcgov/bc-government-organizations-in-github.md:35]: this seems to be a repetition of the above paragraph?

File: docs/use-github-in-bcgov/bc-government-organizations-in-github.md [hunk 17]
```diff
@@ -1,120 +1,102 @@
...
+If any technical errors happen, try to [re-authorize your personal access tokens and/or SSH keys](github-transition-guide.md#resetting-github-keys) so they continue to work with SSO enabled. 
  
-Ministries won’t be charged for upgraded GitHub licenses.  However, ordering IDIRs for contractors may cost and need expense approvals. We have more information about IDIRS on our [IDIR transition guide](github-transition-guide.md#idirs). 
+### SSO timelines for `bcgov-c` 
+
+
+We begin transition of [`bcgov-c`](#bcgov-c) into Enterprise GitHub on **August 7, 2024**. After migration completes, we'll enable SSO for [`bcgov-c`](#bcgov-c) on **August 21, 2024**, using the same process as the SSO enablement for `bcgov`.
   
-### Directions to link your accounts before July 30
+For current B.C. Government private organizations part of the legacy `bcgov-ent` Enterprise implementation, we’ll work with your respective Information Management Branches and Information Systems Branches to identify next steps.
```
Comment (jenreiher-bcgov) [inline docs/use-github-in-bcgov/bc-government-organizations-in-github.md:69]: this content is not related to bcgov-c, so either needs another heading or the heading above needs to be more generic

File: docs/use-github-in-bcgov/bc-government-organizations-in-github.md (hunk 16 above)
Comment (Shivanganii) [reply docs/use-github-in-bcgov/bc-government-organizations-in-github.md:35]: @jenreiher-bcgov Does this sentence need to be here? I have no insight into what it's trying to convey, so if it needs removal, I'm alright with it.

Comment (jenreiher-bcgov) [reply docs/use-github-in-bcgov/bc-government-organizations-in-github.md:35]: let's remove!

File: docs/use-github-in-bcgov/bc-government-organizations-in-github.md [hunk 18]
```diff
@@ -32,39 +32,38 @@ A --> C[bcgov-c]
...
 2. You'll get an email with an invitation to join the **bcgov** GitHub organization. The invitation will be sent to the email address associated with your GitHub account.
 3. Your teammates can add you to your team's repositories.
-
-If any technical errors happen, try to [re-authorize your personal access tokens and/or SSH keys](github-transition-guide.md#resetting-github-keys) so they continue to work with SSO enabled. 
+4. [Re-authorize your personal access tokens and/or SSH keys](github-transition-guide.md#resetting-github-keys).
  
-### SSO timelines for `bcgov-c` and 'bcgov-ent'
+### SSO timelines for `bcgov-c`
 
+We began transition of [`bcgov-c`](#bcgov-c) into Enterprise GitHub on **August 7, 2024**. After migration completes, we'll enable SSO for [`bcgov-c`](#bcgov-c) on **August 21, 2024**, using the same process as the SSO enablement for `bcgov`.
```
Comment (jenreiher-bcgov) [inline docs/use-github-in-bcgov/bc-government-organizations-in-github.md:63]: Aug 27th
